codec-http/src/main/java/io/netty/handler/codec/spdy/SpdyFrameDecoderDelegate.java

```diff
@@ -92,6 +92,15 @@ void readSynStreamFrame(
      */
     void readHeaderBlockEnd();
 
+    /**
+     * Called when an unknown frame is received.
+     *
+     * @param frameType the frame type from the spdy header.
+     * @param flags the flags in the frame header.
+     * @param payload the payload of the frame.
+     */
+    void readUnknownFrame(int frameType, byte flags, ByteBuf payload);
```

Comment:
we can't do this

Reply:
Yes, it break sources, let me adjust it then.